config/locales/en.yml

```diff
@@ -32,6 +32,9 @@ en:
         rwa_number: "RWA Number"
       cart:
         rwa_number: "RWA Number"
+    submit:
+      create: "Submit for approval"
```

Comment:
Should this get scoped more tightly? I can imagine we'll be adding an interface for creating users, etc. where "Submit for approval" isn't a good default text for the submit button.